doc/datomic_differences.md

```diff
@@ -25,8 +25,8 @@ The following functions from the `datahike.api` namespace are different from Dat
 
 - [connect](#connect)
 - [create-database](#create-database)
-- [database-exists?](#database)
-- [db-with](https://cljdoc.org/d/io.replikativ/datahike)
+- [database-exists?](#database-exists) -> not working?
+- [db-with](https://cljdoc.org/d/io.replikativ/datahike) -> all pointing to same place?
```

Comment:
for now we could link to the clj doc entry: https://cljdoc.org/d/io.replikativ/datahike/0.3.6/api/datahike.api#db-with